src/flask_restalchemy/resources/querybuilder.py

```diff
@@ -69,6 +69,8 @@ def build_filter_operator(column_name, request_filter, serializer):
         for field in fields:
             field_name = field.lstrip("-")
             column = getattr(model_class, field_name)
+            if str(column.type) == "VARCHAR":
```

Comment:
Add a const `CASE_INSENSITVE_ORDER_BY_ENABLED = True` and change this to

`if CASE_INSENSITVE_ORDER_BY_ENABLED and str(column.type) == "VARCHAR":`